ignite/cmd/plugin.go

```diff
@@ -387,7 +423,75 @@ func printPlugins() {
 		if p.Error != nil {
 			status = fmt.Sprintf("❌ Error: %v", p.Error)
 		}
+
+		manifest, err := p.Interface.Manifest()
+		if err != nil {
+			panic("Error while loading plugin manifest")
+		}
+
+		var (
+			hookCount = len(manifest.Hooks)
+			cmdCount  = len(manifest.Commands)
+		)
+
+		status = fmt.Sprintf("%s 🪝 %d 💻 %d", status, hookCount, cmdCount)
 		entries = append(entries, []string{p.Path, status})
+
+		session.PrintTable([]string{"Path", "Status"}, entries...)
```

Comment:
Not sure that this should be located inside the loop ;)

Reply:
correct, has been moved.